pytorch_lightning/plugins/training_type/ddp_spawn.py

```diff
@@ -337,7 +337,7 @@ def model_to_device(self):
 
     def pre_backward(self, closure_loss: torch.Tensor) -> None:
         """Run before precision plugin executes backward"""
-        if not self.lightning_module.automatic_optimization and self.model.require_backward_grad_sync:
+        if not self.lightning_module.automatic_optimization:
```

Comment:
this asymmetry between ddp and ddp_spawn was a result of code duplication I guess
I see in DDP we don't have the additional literal :)

Reply:
@awaelchli It was removed recently https://github.com/PyTorchLightning/pytorch-lightning/pull/7970/files :D